docs/spa/index.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: doc
+title: SPA integration guide
+abstract: How to integrate Single Page Webapps with DID.app
+---
+
```

Comment:
Please can we have another line break in here between the first sentence and the abstract?  Presume just add a line in MD>

Reply:
That won't make any difference to the output

Reply:
So is this not possible in the constraints of MD?